src/Manifolds.jl

```diff
@@ -216,6 +225,12 @@ export base_manifold,
     log,
     log!,
     manifold_dimension,
+    mean,
+    mean!,
+    mean_and_var,
```

Comment:
You can import `mean_and_var` and `mean_and_std` from StatsBase.

Reply:
Ah, I looked for it a while and did not see it anywhere.

Reply:
https://github.com/JuliaStats/StatsBase.jl/blob/master/src/moments.jl#L152-L166